index.html

```diff
@@ -0,0 +1,220 @@
+              >See this project<i
+                class="fa fa-long-arrow-right fa-2x"
+                aria-hidden="true"
+              ></i>
+            </a>
+          </div>
+        </div>
+        <div class="info">
+          <img src="img/Macbook-pro-img.jpg" height="270" alt="" />
+          <h2>Project name goes here</h2>
+          <div>
+            <ul class="list">
+              <li class="list-items">HTML/CSS</li>
+              <li class="list-items">Ruby on Rails</li>
+              <li class="list-items">JavaScript</li>
+            </ul>
+            <a class="Project-link" href="#"
+              >See this project<i
+                class="fa fa-long-arrow-right fa-2x"
+                aria-hidden="true"
+              ></i>
+            </a>
+          </div>
+        </div>
+        <div class="info">
+          <img src="img/macbook-pro-img2.jpg" alt="" />
+          <h2>Project name goes here</h2>
+          <div>
+            <ul class="list">
+              <li class="list-items">HTML/CSS</li>
+              <li class="list-items">Ruby on Rails</li>
+              <li class="list-items">JavaScript</li>
+            </ul>
+            <a class="Project-link" href="#"
+              >See this project<i
+                class="fa fa-long-arrow-right fa-2x"
+                aria-hidden="true"
+              ></i>
+            </a>
+          </div>
+        </div>
+      </section>
+      <section class="About">
+        <h2>About</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ea eiusmod
+        </p>
+        <button type="submit">
+          Get my CV <i class="fa fa-download" aria-hidden="true"></i>
+        </button>
+      </section>
+      <div class="Code-lang">
+        <h3>Languages</h3>
+        <ul>
+          <li>JavaScript(ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <div class="Code-lang2">
+        <h3>Languages</h3>
+        <ul>
+          <li>JavaScript(ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <div class="Code-lang3">
+        <h3>Languages</h3>
+        <ul>
+          <li>JavaScript(ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <section class="Info-form">
+        <img src="img/image_geometry_1.png" alt="" />
+        <div class="footh2">
+          <h2>Get started</h2>
+        </div>
+        <div class="footP">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna.
+        </div>
+        <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] So far you have done great to ensure that your form has the action and method 👍 . Though we need to make verification that once an email is sent, the data can be collected by [formspree](https://formspree.io/html) ☺️ . Therefore I suggest you generate a unique `form_id` for the form action attribute 👍 .